lms/djangoapps/certificates/views.py

```diff
@@ -459,6 +459,10 @@ def render_html_view(request):
     )
 
     if active_certificate:
+        # Override the course name with course_title
```

Comment:
Seems like this is really the same as:
`if active_certificate and active_certificate.get('course_title', '')` ?